labrad/backend.py

```diff
@@ -45,6 +47,13 @@ def disconnect(self):
             self._disconnect()
             self.connected = False
 
+    def spawn(self):
+        """Start a new independent backend connection to the same manager."""
+        inst = self.__class__(name=self.name)
```

Comment:
This can't be a classmethod because it needs to copy connection attributes from a particular instance (host, port, etc). Copying the name is not really important; in practice we never use anything other than the default.